src/server/db.js

```diff
@@ -22,8 +22,11 @@ export class InMemory {
    * @param {string} id - The game id.
    * @param {object} store - A Redux store to persist.
    */
-  set(id, store) {
-    this.games.set(id, store);
+  set(id, state) {
```

Comment:
Can these be made [async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) functions to avoid having to manually create promises?

Reply:
I did this with `async` but I got an issue with class methods, I should should find the `babel` config for `async` methods

Reply:
Alternatively, don't use classes at all.